edge/pkg/edged/volume/csi/csi_attacher.go

```diff
@@ -0,0 +1,447 @@
+/*
+Copyright 2017 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
```

Comment:
We need to add changelog if this code is referenced from kubernetes. See this file for example of how to put changelog : https://github.com/kubeedge/kubeedge/blob/master/edge/pkg/edged/edged_getters.go

Reply:
Thanks. Change Log is added.